services/static-webserver/client/source/class/osparc/auth/core/Utils.js

```diff
@@ -51,6 +51,23 @@ qx.Class.define("osparc.auth.core.Utils", {
       return isValid;
     },
 
+    denylistEmailValidator: function(denylist) {
```

Comment:
Q: should this check not be done by the backend? I could disable this validation and bypass it.

Reply:
Ideally, yes.